### PR TITLE
docs: Coalesce release notes for the v9.1.0 release

### PR DESCRIPTION
**Pull request by mhevery**
Merge all of the release notes across all v9.1.0 releases into a single one.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
+* **core:** avoid recursive scope recalculation when TestBed.overrideModule is used ([#35454](https://github.com/angular/angular/issues/35454)) ([0a1a989](https://github.com/angular/angular/commit/0a1a989))
+* **ivy:** remove unused event argument in listener instructions ([#35097](https://github.com/angular/angular/issues/35097)) ([9228d7f](https://github.com/angular/angular/commit/9228d7f))
 
-<a name="9.1.0-rc.1"></a>
-# [9.1.0-rc.1](https://github.com/angular/angular/compare/9.1.0-rc.0...9.1.0-rc.1) (2020-03-23)
 
 
 ### Bug Fixes
 
+* **common:** let `KeyValuePipe` accept type unions with `null` ([#36093](https://github.com/angular/angular/issues/36093)) ([407fa42](https://github.com/angular/angular/commit/407fa42)), closes [#35743](https://github.com/angular/angular/issues/35743)
```
Comment (IgorMinar) [inline CHANGELOG.md]: can you please group these by component? (across all 3 sections: feat, perf, bufix)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
 <a name="9.1.0"></a>
-# [9.1.0](https://github.com/angular/angular/compare/9.1.0-rc.2...9.1.0) (2020-03-25)
+# [9.1.0](https://github.com/angular/angular/compare/9.1.0...9.1.0) (2020-03-25)
```
Comment (IgorMinar) [inline CHANGELOG.md]:
```suggestion
# [9.1.0](https://github.com/angular/angular/compare/9.0.0...9.1.0) (2020-03-25)
```

**Review by IgorMinar — APPROVED** [review]
the rest looks good to me. thanks!

**Comment by IgorMinar** [comment]
btw this should me master&patch - I fixed that.

also, please update the commit message to start with lowercase "c" per our conventions ("Coalesce" => "coalesce")

File: CHANGELOG.md (hunk 2 above)
Comment (mhevery) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 1 above)
Comment (mhevery) [reply CHANGELOG.md]: fixed

**Pull request closed without merging**